payments.md

```diff
@@ -0,0 +1,117 @@
+## BusinessPayment
+
+## Payment collection [/BusinessPayments]
+Payments endpoint allows you to make a Payment anonymously or by logging in. Responses will be returned in JSON format only.
+
+### BusinessPayment
+Payment as a resource represents following information
+
+| Property | Type | As request | As response | Description |
+| :-------------------- | :---------- | :-------------------- | :-------------------- | ------------------------------------------------------------ |
+| id | String | `Mandatory` | `Returned always` | It is the Identifier which is required to track the payment for that particular account.|
+| cardId | String | `Manadatory` | `Returned always` | It is the Identifier for the card that is saved in memcache. |
```

Comment:
Is `card` a different resource? If yes then this property name can be replaced with resource name.